pom.xml

```diff
@@ -128,6 +128,11 @@
 			<version>${plugin.testrunner.version}</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.jenkins-ci.plugins</groupId>
+			<artifactId>hpe-application-automation-tools-common</artifactId>
+			<version>3.0.8-SNAPSHOT</version>
+		</dependency>
 	</dependencies>
```

Comment:
Please don't add general depdencies. it's seem like an old jar that includes many undeed additional files.